docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -60,7 +60,7 @@ metrics based on the tracing data.
   * [Armeria](https://github.com/line/armeria) 0.63.0 -> 0.98.0
   * [Apache Avro](http://avro.apache.org) 1.7.0 - 1.8.x
   * [Finagle](https://github.com/twitter/finagle) 6.44.0 -> 20.1.0  (6.25.0 -> 6.44.0 not tested)
-  * [Brpc-Java](https://github.com/baidu/brpc-java) 2.3.7 -> 2.5.3
+  * [Brpc-Java](https://github.com/baidu/brpc-java) 2.3.7 -> 3.0.5
```

Comment:
I noticed you are changing the repo to https://github.com/baidu/starlight, please update when you finish the changes.